pkg/report/report.go

```diff
@@ -240,7 +240,7 @@ func writeXMLReport(dir, name string, ts *Testsuites) error {
 		return err
 	}
 	xmlStr := string(xDoc)
-	return ioutil.WriteFile(file, []byte(xmlStr), 0644)
```

Comment:
I'm not sure this is a good idea:
- if we keep `0644` and the user wants to restrict the permissions not owner-accessible-only, they can do this with `umask(1)`
- if we change to `0600` but the user does want world-readable, they will need an explicit `chmod` somewhere, which is messy